packages/sdk/config/tsconfig.json

```diff
@@ -2,33 +2,24 @@
   "extends": "../../../tsconfig.base.json",
   "compilerOptions": {
     "allowJs": true,
-    "lib": [
-      "DOM",
-      "ESNext"
-    ],
-    "outDir": "./dist/types"
```

Comment:
I've noticed this removed in a bunch of places, is that because it's correctly pulling it from the project.json now?